aemedge/blocks/content-list/content-list.js

```diff
@@ -5,6 +5,8 @@ import PictureCard from '../../libs/pictureCard/pictureCard.js';
 import Card from '../../libs/card/card.js';
 import Button from '../../libs/button/button.js';
 import { formatDate } from '../../scripts/utils.js';
+import { getAuthorEntries } from '../author-profiles/author-profiles.js';
```

Comment:
We again can't go that route of injecting one block as dependency in another as we have difficulty is cleaning up article-list and making content-list independent. This function can be put in utils.js and used in multiple places. 
@mhaack WDYT?

Reply:
@saurabh-khare : Corrected. Code has been refactored into /libs/profile/profile.js